massa-api/src/public.rs

```diff
@@ -832,8 +832,25 @@ impl MassaRpcServer for API<Public> {
                 .collect()
         };
 
+        // Compute a limit (as a slot) for deferred credits as it can be quite huge
+        let bound_ts = MassaTime::now()
+            .checked_add(MassaTime::from_millis(
+                60 * 24 * 60 * 60 * 1000, // 60 Days, 24 hours, 60 minutes, 60 seconds, 1000 milliseconds
```

Comment:
no magic numbers, let's make it an API config file parameter

Reply:
Done